src/renderer/features/network/NetworkSelector/ui/NetworkSelector.tsx

```diff
@@ -118,7 +86,7 @@ export const NetworkSelector = ({
             <div ref={ref} className={OptionsContainerStyle}>
               <Listbox.Options className="max-h-64 overflow-y-auto overscroll-contain">
                 {availableNodes.map((data) => {
```

Comment:
I suppose we can do this:
```javascript
 {availableNodes.map(({ type, node }) => {
```

Reply:
But we still use `data` itself for Listbox.Option value